mkdocs.yml

```diff
@@ -87,151 +87,122 @@ plugins:
             - /limitador-server/sandbox/*
             - /LICENSE
         - name: limitador-operator
-          import_url: 'https://github.com/kuadrant/limitador-operator?edit_uri=/blob/main/&branch=main'
+          import_url: "https://github.com/kuadrant/limitador-operator?edit_uri=/blob/main/&branch=main"
           imports:
             - /README.md
             - /doc/*
         - name: architecture
-          import_url: 'https://github.com/kuadrant/architecture?edit_uri=/blob/main/&branch=main'
+          import_url: "https://github.com/kuadrant/architecture?edit_uri=/blob/main/&branch=main"
           imports:
             - /rfcs/*
             - /docs/*
         - name: api-quickstart
-          import_url: 'https://github.com/kuadrant/api-quickstart?edit_uri=/blob/main/&branch=main'
+          import_url: "https://github.com/kuadrant/api-quickstart?edit_uri=/blob/main/&branch=main"
           imports:
             - /README.md
             - /images/*
         - name: kuadrantctl
-          import_url: 'https://github.com/kuadrant/kuadrantctl?edit_uri=/blob/main/&branch=main'
+          import_url: "https://github.com/kuadrant/kuadrantctl?edit_uri=/blob/main/&branch=main"
           imports:
             - /README.md
             - /doc/*
         - name: dns-operator
-          import_url: 'https://github.com/kuadrant/dns-operator?edit_uri=/blob/main/&branch=main'
+          import_url: "https://github.com/kuadrant/dns-operator?edit_uri=/blob/main/&branch=main"
           imports:
             - /README.md
             - /docs/*
             - /config/samples/*
 nav:
-  - 'Overview': index.md
```

Comment:
v hard to read this diff with all the formatting stuff. any chance of a cleaner diff when unhold-ed?